features/attached_enable.feature

```diff
@@ -273,6 +273,7 @@ Feature: Enable command behaviour when attached to an UA subscription
         Then stdout matches regexp:
         """
         cis          +yes      +disabled +Center for Internet Security Audit Tools
+        esm-apps     +yes      +enabled  +UA Apps: Extended Security Maintenance \(ESM\)
```

Comment:
Is it expected that esm-apps will be on this list, even without the `--all`, being a beta service?

Reply:
Yep, When a beta service is enabled, then it will show up in this list with or without `--all`

Reply:
Ack!